scripts/composer/class-mitcomposerscripts.php

```diff
@@ -61,7 +61,7 @@ public static function multidev_create_syntax( $event ): void {
 	 */
 	public static function multidev_search_replace_syntax( $event ): void {
 		$multidev = self::multidev_name( $event );
-		$terminus_command = "terminus remote:wp mitlib-wp-network.$multidev -- search-replace live-mitlib-wp-network.pantheonsite.io $multidev-mitlib-wp-network.pantheonsite.io --url=live-mitlib-wp-network.pantheonsite.io --network";
+		$terminus_command = "terminus remote:wp mitlib-wp-network.$multidev -- search-replace libraries.mit.edu $multidev-mitlib-wp-network.pantheonsite.io --url=libraries.mit.edu --network";
```

Comment:
Can you clarify what this is fixing? I'm assuming behavior has been weird since launch and we've needed to manually re-run the search-replace and this will possible make moving data between tiers less manual?

Reply:
This is the reminder about what search-replace command we should be running - it doesn't actually run the command.

The current output:
```bash
% composer multidev-search-replace-syntax pw-106
> MitComposerScripts::multidev_search_replace_syntax
terminus remote:wp mitlib-wp-network.pw-106 -- search-replace live-mitlib-wp-network.pantheonsite.io pw-106-mitlib-wp-network.pantheonsite.io --url=live-mitlib-wp-network.pantheonsite.io --network
```

The problem is that this command won't do what we need now that we've got our domain in place. So every time we get this reminder, we need to remember to update the `live-mitlib-wp-network.pantheonsite.io` value.

This PR changes the output to be something that is accurate:
```bash
% composer multidev-search-replace-syntax pw-106
> MitComposerScripts::multidev_search_replace_syntax
terminus remote:wp mitlib-wp-network.pw-106 -- search-replace libraries.mit.edu pw-106-mitlib-wp-network.pantheonsite.io --url=libraries.mit.edu --network
```

Reply:
Oh right! I totally forgot about that. Thanks for the reminder.